indico/modules/events/registration/templates/display/_registration_summary_blocks.html

```diff
@@ -141,11 +141,20 @@
                             <td class="text-right">{{ item.price }} {{ currency }}</td>
                         </tr>
                     {% endfor %}
+                    {% if registration.registration_form.base_price %}
+                        <tr>
```

Comment:
Just to highlight the fact that these fees don't come from what they chose in the registration. I think some visual clue would be helpful.